src/test/rustdoc/intra-doc/prim-associated-traits.rs

```diff
@@ -0,0 +1,6 @@
+use std::{num::ParseFloatError, str::FromStr};
+
+/// Uses the rules from [`f64::from_str()`].
```

Comment:
This test doesn't do much without a check like this:

```suggestion
// @has 'doc/prim_associated_traits/struct.Number.html' '//a[@href="https://doc.rust-lang.org/nightly/std/primitive.f64.html#method.from_str"]' 'f64::from_str()'
/// Uses the rules from [`f64::from_str()`].
```